labs/optimizely-data-cli/README.md

```diff
@@ -0,0 +1,162 @@
+
+## Installation 
+
+`oevents` is a bash script. To run it, you'll need to make sure it's executable.  Assuming `oevents` is in your working directory, you can do this with the following command:
+
+```sh
+$ chmod +x oevents
+```
+
+To run `oevents`, you can specify the script's path explicitly like this (again, assuming `oevents` is in your working directory):
+
+```sh
+$ ./oevents help
+```
+
+Or you can add it to a directory in your `PATH` environment variable.  
+
+```sh
+$ sudo cp oevents /usr/local/bin/
+$ oevents help
+```
+
+## Get Help
+
+For usage instructions, use the `help` command:
+
+```sh
+$ oevents help
+```
+
+## Authenticating
+
+Optimizely Enriched Event data is served via [Amazon S3](https://aws.amazon.com/s3/). `oevents` uses the [Amazon AWS CLI](https://aws.amazon.com/cli/) to interact with S3, which requires authentication.  You can authenticate to AWS in three ways:
+
+1. (Recommended) Pass your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) to `oevents` via the `OPTIMIZELY_API_TOKEN` environment variable:
+
+    ```sh
+    $ export OPTIMIZELY_API_TOKEN=<token>
+    $ oevents ls
+    ```
+2. Pass your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) and [Optimizely Account ID](https://help.optimizely.com/Account_Settings/Account_name%2C_ID%2C_and_multi-account_login_in_Optimizely_X) via the `--token` and `--account-id` command line arguments:
+
+    ```sh
+    $ oevents ls --token <token> --account-id 12345
+    ```
+3. Provide your AWS credentials directly. See the [AWS user guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for instructions.
+
+If you chose methods 1 or 2, `oevents` will acquire temporary AWS credentials using the [Optimizely Authentication API](https://docs.developers.optimizely.com/optimizely-data/docs/authentication-api).
+
+
+**Note:** If you have an Optimizely Personal Access Token, you can use the `oevents auth` command to acquire temporary AWS credentials that may be used directly with the AWS CLI or client libraries in your ETL pipeline:
+
+```sh
+  $ oevents auth --token <optimizely personal access token>
+
+  export AWS_ACCESS_KEY_ID=<key id>
+  export AWS_SECRET_ACCESS_KEY=<secret access key>
+  export AWS_SESSION_TOKEN=<session token>
+  export AWS_SESSION_EXPIRATION=1594953226000
+  export S3_BASE_PATH=s3://optimizely-events-data/v1/account_id=12345/
+```
+
+## Exploring your Enriched Event data
+
+[decisions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-decisions
+[conversions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-conversions
+
+Enriched Events are partitioned into two top-level datasets, [decisions] (`type=decisions`) and [conversions] (`type=events`).  Each of these datasets is partitioned by date and experiment (for decisions) or event type (for conversions).  
```

Comment:
Has there been any thought in telling the export team to export `type=conversions` to centralize language?

Reply:
No, unfortunately I think that ship has sailed for v1.  Maybe for v2? cc @annabarr